packages/-ember-data/tests/integration/inverse-test.js

```diff
@@ -111,41 +120,41 @@ module('integration/inverse_test - inverseFor', function(hooks) {
   });
 
   test('Returns null if inverse relationship it is manually set with a different relationship key', function(assert) {
-    Job.reopen({
+    job.reopen({
```

Comment:
we should avoid reopens of models. Either register a new class or extend the original class. It might be that these classes aren't as "dry" as they seem at first and should just be created and registered uniquely in each test.